homeassistant/components/reolink/camera.py

```diff
@@ -18,10 +19,11 @@
 async def async_setup_entry(
     hass: HomeAssistant,
     config_entry: ConfigEntry,
-    async_add_devices: AddEntitiesCallback,
+    async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up a Reolink IP Camera."""
-    host: ReolinkHost = hass.data[DOMAIN][config_entry.entry_id].host
+    reolink_data: ReolinkData = hass.data[DOMAIN][config_entry.entry_id]
+    host: ReolinkHost = reolink_data.host
```

Comment:
`ReolinkData` is already typed.

```suggestion
    host = reolink_data.host
```